doc/requirements.txt

```diff
@@ -1,4 +1,5 @@
 docutils==0.16
+mthree
```

Comment:
Minor: would this be required to build the docs?

Reply:
Yes it is necessary exactly.